doc/client_lua_api.md

```diff
@@ -787,6 +787,8 @@ Call these functions only at load time!
       extra arguments and return the result
 * `fgettext(string, ...)` : returns string
     * same as fgettext_ne(), but calls core.formspec_escape before returning result
+* `minetest.pointed_thing_to_face_pos(placer, pointed_thing)`: returns a table
```

Comment:
@MarkuBu `deserialize` returns a table for which we don't know the keys beforehand, while this new function does, so your example is irrelevant. I think @rubenwardy's comment is correct.